atomic_reactor/utils/remote_host.py

```diff
@@ -672,7 +670,12 @@ def lock_resource(self, prid: str) -> Optional[LockedResource]:
 
         resources = []
         for host in self.hosts:
-            available_slots = host.available_slots()
+            try:
+                available_slots = host.available_slots()
+            except Exception as ex:
+                # Specific exceptions should be handled in nested methods
+                logger.warning("%s: unable to get available slots: %s", host.hostname, ex)
+                available_slots = []
```

Comment:
Maybe `continue` straight away? Otherwise logs will say this
```
<host>: unable to get available slots: <error>
<host>: no available slots
```
^ "no available slots" is not true, we don't know that

Reply:
I removed the log to avoid confusion

Reply:
Don't we lose the info about hosts which really don't have any available slots?

Reply:
fixed it. Logging both cases in different conditions